photon_installer/isoInstaller.py

```diff
@@ -77,24 +88,29 @@ def __init__(self, options):
 
     def _load_ks_config(self, path):
         """kick start configuration"""
-        if path.startswith("http://"):
+
+        if path.startswith("http://") and not self.insecure_installation:
+            raise Exception("Refusing to download kick start configuration from non-https URLs. \
+                            \nPass insecure_installation=1 as a paramameter when giving http url in ks.")
+
+        if path.startswith("https://") or path.startswith("http://"):
```

Comment:
Can we use 'path.startswith("http"):' here?

Reply:
No, we need to use both http& https, ks_config can be downloaded for both
insecure_installation=0  https(with correct ssl certificate required)
inseucre_installation=0 http(exception)
insecure_installation=1 https,http both work

Reply:
yeah, i get it .. but both startswith "http"

Reply:
we need correct path, someone can give httpq://  for eg.  Perhaps we should throw error in case of other path

Reply:
using path like httpq will throw error from json wrapper.